settings.gradle.kts

```diff
@@ -78,25 +78,20 @@ gradle.rootProject {
   versions.forEach { k, v -> prj.extra[k.toString()] = v }
 }
 
-include("errorprone")
-
-include("checkstyle")
-
-include("ide-integration")
-
-include("publishing")
-
-include("jandex")
-
-include("jacoco")
-
-include("protobuf")
-
-include("reflection-config")
-
-include("smallrye-openapi")
-
-include("spotless")
+listOf(
+    "errorprone",
+    "checkstyle",
+    "ide-integration",
+    "publishing",
+    "jandex",
+    "jacoco",
+    "protobuf",
+    "reflection-config",
+    "smallrye-openapi",
+    "spotless",
+    "dependency-declarations"
```

Comment:
wondern:
where does this come from?
i would have thought these names refer to top level folders in the repo
or are these references to projects and the last one we created on the fly?

Reply:
Those are just project names - not related to the project directories. See Spark-extensions projects in Nessie - same project directory shared for multiple projects.